extensions/legit/porcelain.lisp

```diff
@@ -433,6 +666,13 @@ M	src/ext/porcelain.lisp
     (str:starts-with-p hash root)))
 
 (defun rebase-interactively (&key from)
+  (case *vcs*
+    (:git (git-rebase-interactively :from from))
+    (:hg (error "Interactive rebase not implemented for Mercurial"))
+    (:fossil (error "No interactive rebase for Fossil."))
+    (t (error "Interactive rebase not available for this VCS: ~a" *vcs*))))
```

Comment:
Basically, ERROR is treated as an editor bug when the command loop catches it.
Wouldn't it be better to use editor-error?

Reply:
I wanted to follow your suggestion and use `lem:editor-error`, but then this adds coupling, again, and I would prefer to avoid it. I found how to load Lem in Emacs, but I am hesitant. `porcelain` can be a package on its own… unless it makes no sense, since all commands that use it are defined on Lem's side. 

I'll think this through a bit more.

Reply:
OK I do it by simply using uiop:symbol-call again.